spec/luassert_helper.lua

```diff
@@ -1,8 +1,27 @@
 local busted = require('busted')
 local assert = require('luassert.assert')
+local spy = require('luassert.spy')
+local stub = require('luassert.stub')
 local util = require ('luassert.util')
 local say = require('say')
 
+local original_stub_new = stub.new
+local original_spy_on = spy.on
+
```

Comment:
The PR looks good. This was tricky :+1:

I think we should add a comment here that explains why overriding these two functions is needed. We should mention that we are reverting stubs and spies on every tests, but if we double-stub/spy, then only the latest is reverted.